Python/setup.py

```diff
@@ -21,7 +21,7 @@ def readme():
         return f.read()
 
 setup(name='ibmcloudsql',
-      version='0.3.17',
+      version='0.4.0',
       python_requires='>=2.7, <4',
```

Comment:
@torsstei : maybe you should also update `python_requires` to require Python 3.x+?

Reply:
Not sure if we should disallow it explicitly.

Reply:
I see.